python-lib/ontology_tagger.py

```diff
@@ -108,14 +121,23 @@ def _tokenize_keywords(
             List: The tokenized keywords.
```

Comment:
The `tags` argument is not in the docstring. What is it ? Apparently it should have the same number of elements as `keywords`, should write this down